public/content/documentation/web/component/strikethrough-content.md

```diff
@@ -6,75 +6,94 @@ How to test a strikethrough element
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a strikethrough element
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+      - Arrow: Browses the content
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+      - Swipe: The content makes sense and is in logical order
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - Group: It typically contains the name and primary navigation of the website
+      - Name: The content makes sense and is in logical order
```

Comment:
Same as above here; could we put "a" ahead of "logical order" so it reads a bit clearer?